specs/spec-2-election-ui.md

```diff
@@ -0,0 +1,200 @@
+
+The following is to provide context for the electing the NDC Congressional Bodies, which are the House of Merit, Transparency Commission, and Council of Advisors. Voting is conducted by verified humans in the NDC (denoted as members of the Voting Body).
+
+The overall governance flow:
+
+1. **Nomination process**, which involves a self vote (based off of kudos) and accompanying attestation and a kudo + attestation by another voting body members
+2. **Election process** (the scope of this RFP), which involves voting body members having x amount of credits per congressional body and congressional 
+3. **Congress in session**: when the elected congressional members make decisions during their bi-annual tenure in their body including passing setup packages, approving budget of constellations etc. 
+
+We are assuming Nomination->Election ->Congressional Onboarding are done manually, (ex; by the governance working group) as opposed to automatically updated by cross contract calls. Each part of the process which is not manually handled will be on-chain. Automation will be worth building once we've proven the system works via manual effort. 
+
+Mechanisms of Nomination are out of scope as are mechanisms of assigning congress seats. We do need a way to clearly communicate who has been elected to congress seats.
+
+## Use Cases
+
+<!-- Who is using this and what does it do? Leave "how" to the design/tech specification sections. If the solution doesn't help these people achieve their goal then the project isn't successful, even if it fills all of the specified technical requirements. -->
+<!-- List the different groups of people that will use this widget and what each one can specifically do. If more clarity is needed you can include an actor/action matrix. List all users and their types. List the actions each collective will take individually. User profile description. Describe which action is done by which actors. Feel free to use a table format or provide your own graphics or bundle the description in the Actions section. A "swimlane process chart" often works well here. -->
+
+Actions are broken down by actors:
+
+- Voter
+- Candidate
+- Watchdog (no IAH credentials)
+
+These user stories illustrate the needs and desires of the actors rather than perscribing how the actors will achieve their goals.
+
+### Voter
+
+#### Outside of Election UI but still in BOS
+
+- I want to be notified when the election opens and notified when the election is about to close directly on BOS [notification widget].
+- I want to know about upcoming elections and their dates. 
```

Comment:
Let's maybe specify that we are expecting a kind of a dashboard with:

* section about upcoming elections (should be broken down by a house? or all combined?)
* dashboard with latest news.

Reply:
MVP won't have a separate dashboard. Upcoming elections can be discovered two ways in the current interface: (1) Congress/Time selector (2) List of upcoming elections under the current election status graphic.

Reply:
ok, then let's describe it in the spec.